### PR TITLE
Updated URL

### PR DESCRIPTION
**Pull request by Phinetwork**
Updated URL

### CODE REVIEW

File: _data/icons/aac.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [
     {
-	"url":"ipfs://QmRUrz4dULaoaMpnqd8qXT7ehwz3aaqnYKY4ePsy7isGaF",
+	"url":"https://ipfs.io/ipfs/QmV6z97Cr9t8j7pP1vWfp9ypioig5NcS9RLU5S2cwwMfDM?filename=logo.png",
```
Comment (ligi) [inline _data/icons/aac.json:3]:
```suggestion
	"url":"ipfs://QmV6z97Cr9t8j7pP1vWfp9ypioig5NcS9RLU5S2cwwMfDM",
```

**Review by ligi — APPROVED** [review]

**Comment by Phinetwork** [comment]
This was supposed to Be logo Updated for phi.json not aac.json added logo for Phi.json

**Comment by ligi** [comment]
so the filename is wrong then - please fix

**Comment by Phinetwork** [comment]
Cancel this is not right file for this image image link is for phi.json

**Pull request closed without merging**